stream-transformation-test/sample-transformations/src/main/java/uk/gov/sample/event/transformation/SampleCustomActionAndDeactivateTransformation.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.sample.event.transformation;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static uk.gov.justice.tools.eventsourcing.transformation.api.Action.DEACTIVATE;
+import static uk.gov.justice.tools.eventsourcing.transformation.api.Action.NO_ACTION;
+
+import uk.gov.justice.services.core.enveloper.Enveloper;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+import uk.gov.justice.tools.eventsourcing.transformation.api.Action;
+import uk.gov.justice.tools.eventsourcing.transformation.api.EventTransformation;
+import uk.gov.justice.tools.eventsourcing.transformation.api.annotation.Transformation;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+@Transformation
+public class SampleCustomActionAndDeactivateTransformation implements EventTransformation {
+
+    private static final List<String> EVENTS_TO_DEACTIVATE = newArrayList(
+            "referencedata.result.definition-uploaded",
+            "referencedata.result.definition-added",
+            "referencedata.result.prompt-uploaded",
+            "referencedata.result.prompt-added",
+            "referencedata.result.prompt-keyword-synonym-uploaded",
+            "referencedata.result.prompt-keyword-synonym-added",
+            "referencedata.result.prompt-fixedlist-uploaded",
+            "referencedata.result.prompt-fixedlist-added",
+            "referencedata.result.definition-keyword-synonym-uploaded",
+            "referencedata.result.definition-keyword-synonym-added"
+    );
+
+    private static final String EARLIER_RENAMED_EVENT_NAME_ENDS_WITH = ".archived.1.9.release";
```

Comment:
What is this class actually showing ? IF we already have a deactivate sample.

Reply:
If this is causing confusion then I can remove it. This is just another example of transformation where we are renaming the event name. I wanted to test it that it still works with multiple actions in apply method